jdaviz/configs/default/plugins/export/export.py

```diff
@@ -259,6 +280,15 @@ def export(self, filename=None, show_dialog=None):
                 raise NotImplementedError(f'Subset can not be exported - {self.subset_invalid_msg}')
             self.save_subset_as_region(selected_subset_label, filename)
 
+        elif len(self.dataset.selected):
+            filetype = self.dataset_format.selected
+            if not filename.endswith(filetype):
+                filename += f".{filetype}"
+            if self.data_invalid_msg != "":
+                raise NotImplementedError(f"Data can not be exported - {self.data_invalid_msg}")
+            if isinstance(self.dataset.selected_obj, Spectrum1D):
+                self.dataset.selected_obj.write(Path(filename), overwrite=True)
+

```

Comment:
Let's make sure we don't raise a success snackbar message in cases where `write` is not called (although these cases - like images that @cshanahan1 brought up - should either be supported OR included in the `data_invalid_msg` logic).

```suggestion
            if isinstance(self.dataset.selected_obj, Spectrum1D):
                self.dataset.selected_obj.write(Path(filename), overwrite=True)
            else:
                raise NotImplementedError(f"exporting {self.dataset.selected} not supported")

```